### PR TITLE
fix: Fix UI captions icon state

### PR DESCRIPTION
**Pull request by kartikeyshaurya**
fix: Fixed minor bug #4358 
Part of #4358  fixed the code 
Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
not work as expected)
- [ ] This change requires a documentation update

Checklist:

- [x] I have signed the Google CLA https://cla.developers.google.com/
- [x] My code follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [x] I have verified my change on multiple browsers on different platforms
- [x] I have run ./build/all.py and the build passes
- [x] I have run ./build/test.py and all tests pass

Reviewed-by: @anshgo01yal 
Refs: #4358

### CODE REVIEW

**Review by anshgo01yal — DISMISSED** [review]

**Review by joeyparrish — APPROVED** [review]

File: ui/text_selection.js [hunk 1]
```diff
@@ -111,11 +111,11 @@ shaka.ui.TextSelection = class extends shaka.ui.SettingsMenu {
   onCaptionStateChange_() {
     if (this.player.isTextTrackVisible()) {
       this.icon.textContent =
-          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS_OFF;
+          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS;
       this.button.ariaPressed = 'true';
     } else {
       this.icon.textContent =
-          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS;
+          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS__OFF;
```
Comment (avelad) [inline ui/text_selection.js:118]: I think that the name is erroneous

Comment (joeyparrish) [reply ui/text_selection.js:118]: And that's why we have compilers and linters and tests!  :grin: 

@kartikeyshaurya, please take a look and try `python build/check.py` to confirm your changes.

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Fails linter/compiler checks

**Review by joeyparrish — APPROVED** [review]

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Please revert your changes to the build-and-test workflow

**Comment by joeyparrish** [comment]
> Please revert your changes to the build-and-test workflow

And/or rebase.

**Comment by kartikeyshaurya** [comment]
@joeyparrish  
I have made the Changes according to which you asked and also there is no Fails linter/compiler checks . Kindly look into it , as per above failing issue , i think there is drive issue in the testing tool 
image

**Comment by joeyparrish** [comment]
I'm not referring to the test failure.  We have some known flakiness in our tests/infra.

Please look here: https://github.com/shaka-project/shaka-player/pull/4384/files#diff-a551b322da0f9fe92abd1ec41e43c58dc5d138c0000430395e66f0c581387cd2

You have changes to a build workflow in your commit for some reason.  Can you please revert those or rebase to main or both?

**Review by avelad — APPROVED** [review]

**Comment by avelad** [comment]
@joeyparrish can you review it? Thanks!

**Review by joeyparrish — APPROVED** [review]

**Pull request merged**